docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -64,6 +64,52 @@ The dashboard is no longer accessible, even with the link, until you make it sha
 
 The link no longer works. You must create a new public URL as in [Make a dashboard public](#make-a-dashboard-public).
 
+## Email Sharing
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud).
+
+Email sharing allows you to share your public dashboard with only specific people via email, instead of having it accessible to anyone with the url.
```

Comment:
```suggestion
Email sharing allows you to share your public dashboard with only specific people via email, instead of having it accessible to anyone with the URL.
```